data/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Your Edge App name</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```

Comment:
We should probably add something that users the metadata. Like

`Hello World, I'm $SCREEN_NAME`

Reply:
Sure, will change that a bit later